src/main/java/LacksDataBaseProject/CRUDRepository.java

```diff
@@ -0,0 +1,11 @@
+package LacksDataBaseProject;
+
+import java.util.List;
+
+public interface CRUDRepository<T extends Object> {
```

Comment:
Tutaj zdecydowanie lepiej byłoby zrobić coś w stylu: <T extends BaseEntity> czy coś w tym stylu, każdy obiekt w Javie dziedziczy po klasie Object więc tutaj to możecie wszystko dać, a wy byście pewnie chcieli stworzyć repo tylko pod określoną grupę obiektów :)

Reply:
Tak tylko, że te 3 klasy obiektów dla których tworzę repozytoria nie mają ze sobą nic wspólnego... więc ALBO: mogę dorobić sztucznie jakąś klasę "BaseEntity" z jakąś 1 wspólną zmiennną  typu: baseId i będzie: CRUDRepository<T extends BaseEntity>  ALBO: mogę zrobić 3 osobne interfejsy CRUDUserRepo... LacksRepo... itd. tyle, że wtedy gubię tu polimorfizm. To jak będzie lepiej wg. Ciebie?

Reply:
Moim zdaniem najlepiej by było zrobić jakąś klasę BaseEntity tak by można było zawęzić zakres Twojego repo :)